CanvasColorSpaceProposal.md

```diff
@@ -95,13 +96,16 @@ partial dictionary CanvasRenderingContext2DSettings {
 };
 </pre>
 
-* All input colors (e.g, ``fillStyle`` and ``strokeStyle``) follow the same interpretation as CSS color literals, regardless of the canvas color space.
-* Images drawn to a canvas are converted from their native color space to the backing storage color space of the canvas.
+2D Canvases are color managed.
+* All inputs that are drawn to a canvas have specific color space
```

Comment:
"have a specific color space" - and end sentence with period - but see comment below.

Reply:
Fixed